common/route/awooter/src/partition.rs

```diff
@@ -1376,14 +1373,16 @@ impl PipSelector {
             .flat_map(|pos| pips.get(&pos))
             .flat_map(|vec| vec.iter())
             .filter_map(|pip| {
-                if !ctx.pip_avail_for_net(*pip, raw_net) {
+                // checkPipAvailForNet should really have its NetInfo arg annotated as const,
+                // thus this dubious-looking cast and nonsense unsafety
+                if unsafe { !ctx.pip_avail_for_net(*pip, raw_net as *const _ as *mut _) } {
```

Comment:
You mean like in c62a947a2804c3027471b20815c7544e0fd698d0?

Reply:
...as it turns out, yes!

Reply:
Since it's not `const T& const`, that only gets us to matching `*const NetInfo` according to cxx. I still removed the lamentation from the comments since a cast to `*const _` is accordingly less peculiar.